### PR TITLE
get data dynamically

### PR DESCRIPTION
**Pull request by Muniralsharif**
change dropdown to display from the database 
**************************** relates #2 **************************************

### CODE REVIEW

File: client/src/Components/Search/index.js [hunk 1]
```diff
@@ -11,9 +11,27 @@ class Search extends Component {
 		location: '',
 		pharmacy: '',
 		searchButtonClicked: false,
-		loading: false
+		loading: false,
+		dropdownlocation:["0"],
+		dropdownPharmacyname:["0"],
```
Comment (Hadeel-Salamin) [inline client/src/Components/Search/index.js]: you can just add an empty array

File: client/src/Components/Search/index.js [hunk 2]
```diff
@@ -11,9 +11,27 @@ class Search extends Component {
...
+
 	}
+  componentDidMount() {
+	let pharmacyNames=[];
```
Comment (Hadeel-Salamin) [inline client/src/Components/Search/index.js]: you can just declare these arrays on the next lines where you started using them

File: client/src/Components/Search/index.js [hunk 3]
```diff
@@ -11,9 +11,27 @@ class Search extends Component {
...
+  let uniqeArray=[];
+		axios.get(`/pharmacyDropDown`).then(({ data }) => {
+			uniqeArray=data.data;
+			pharmacyNames=uniqeArray.map(item => item.pharmacyname);
+			pharmacyLocation=uniqeArray.map(item => item.location);
+			pharmacyNames=[...new Set(pharmacyNames)]
+			pharmacyLocation=[...new Set(pharmacyLocation)]
+				this.setState({ dropdownlocation:pharmacyLocation,dropdownPharmacyname:pharmacyNames })
+
+				console.log("hello ", data)
```
Comment (Hadeel-Salamin) [inline client/src/Components/Search/index.js]: remove console.log

File: client/src/Components/Search/index.js [hunk 4]
```diff
@@ -11,9 +11,27 @@ class Search extends Component {
...
+		dropdownlocation:["0"],
+		dropdownPharmacyname:["0"],
+
 	}
+  componentDidMount() {
+	let pharmacyNames=[];
+	let pharmacyLocation=[];
+  let uniqeArray=[];
+		axios.get(`/pharmacyDropDown`).then(({ data }) => {
+			uniqeArray=data.data;
```
Comment (Hadeel-Salamin) [inline client/src/Components/Search/index.js]: the variable name is not clear

File: client/src/Components/Search/index.js [hunk 5]
```diff
@@ -11,9 +11,27 @@ class Search extends Component {
...
-		loading: false
+		loading: false,
+		dropdownlocation:["0"],
+		dropdownPharmacyname:["0"],
+
 	}
+  componentDidMount() {
+	let pharmacyNames=[];
+	let pharmacyLocation=[];
+  let uniqeArray=[];
```
Comment (Hadeel-Salamin) [inline client/src/Components/Search/index.js]: I would use const for this array as you are not modifying it anywhere else

File: client/src/Components/Search/index.js [hunk 6]
```diff
@@ -92,21 +112,24 @@ class Search extends Component {
...
+					)}
 				</select>
 				<select
 					value={pharmacy}
 					name="By pharmacy"
 					onChange={this.searchByPharmacy}
 					className="By-Pharmacy"
-				>
+					>
 					<option value="">By Pharmacy</option>
```
Comment (Hadeel-Salamin) [inline client/src/Components/Search/index.js]: add key value

File: server/controllers/index.js [hunk 7]
```diff
@@ -16,5 +17,6 @@ router.post('/api/login', login.get)
 router.post('/api/pharmacy/medicine', auth, addMedicine)
 router.patch('/api/pharmacy/medicine/:medicineid', auth, updateMedForm)
 router.get('/api/logout', logout.get)
+router.get('/pharmacyDropDown', getpharmacyInfo)
```
Comment (Hadeel-Salamin) [inline server/controllers/index.js]: use '/api/... for all endpoints

**Review by YousefQawasmeh — APPROVED** [review]
nice work :)

**Review by someyaaltous — APPROVED** [review]

**Review by Hadeel-Salamin — APPROVED** [review]

**Pull request merged**